app/webpacker/packs/internal.js

```diff
@@ -0,0 +1,16 @@
+import { initAll } from 'govuk-frontend';
+import { enhanceSelectElement } from 'accessible-autocomplete';
+import 'trix';
+
+initAll();
+
+document.body.className = document.body.className
+  ? document.body.className + ' js-enabled'
```

Comment:
What's this used for?

Reply:
https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#add-the-javascript-file-to-your-html

For the GovUK JS to work

Reply:
Ah ok, may be worth adding a comment in that case; its the kind of thing that someone would likely remove at some point as theres no direct reference to it in the app